src/platform-includes/replay/privacy-configuration/javascript.mdx

```diff
@@ -1,13 +1,13 @@
-| maskAllInputs | boolean                  | `true`                                   | Mask values of `<input>` elements. Passes input values through `maskFn` before sending to server.                                                                                                 |
-| block         | string[]                 | `.sentry-block, [data-sentry-block]`     | Redact all elements that match the DOM selector(s). See [Blocking](#blocking) section for an example. Note that any configured selectors will be in _addition_ to the defaults.                   |
-| blockAllMedia | boolean                  | `true`                                   | Block _all_ media elements (`img`, `svg`, `video`, `object`, `picture`, `embed`, `map`, `audio`).                                                                                                 |
-| ignore        | string[]                 | `.sentry-ignore, [data-sentry-ignore]`   | Ignores all events on the matching input fields. See [Ignoring](#ignoring) above for an example.                                                                                                  |
-| maskFn        | (text: string) => string | `(text) => '*'.repeat(text.length)`      | Function to customize how text content is masked before sending to server. By default, masks text with `*`.                                                                                       |
-| unblock       | string[]                 | `.sentry-unblock, [data-sentry-unblock]` | Don't redact any elements that match the DOM selectors. Used to unblock specific media elements that are blocked with `blockAllMedia`. This doesn't affect sensitive elements such as `password`. |
-| unmask        | string[]                 | `.sentry-unmask, [data-sentry-unmask]`   | Unmask all elements that match the given DOM selectors. Used to unmask specific elements that are masked with `maskAllText`.                                                                      |
+| key           | type               | default                                  | description                                                                                                                                                                                       |
+| ------------- | ------------------ | ---------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| mask          | `string[]`         | `.sentry-mask, [data-sentry-mask]`       | Mask all elements that match the given DOM selectors. See [Masking](#masking) section for an example. Note that any configured selectors will be in _addition_ to the defaults.                   |
```

Comment:
Can we make this also valid js (i.e. strings)? Same for the `block`, `unblock`, `ignore` and `unmask` variables below.